code/AdobeEdge.brs

```diff
@@ -570,15 +570,10 @@ function _adb_task_node_EventProcessor(internalConstants as object, task as obje
                 return
```

Comment:
We can move this check  if (not m._hasValidConfig())

to processRequests() where we check for valid ecid and configid